spec/griddler/email_spec.rb

```diff
@@ -432,6 +432,15 @@ def email_with_params(params)
     expect(headers[header_name]).to eq header_value
   end
 
+  it 'handles a hash being submitted' do
```

Comment:
I am happy to change it, but all the other tests in this file use single quoted string.